src/wp-includes/option.php

```diff
@@ -653,6 +648,7 @@ function add_option( $option, $value = '', $deprecated = '', $autoload = 'yes' )
 				$deprecated_keys[ $option ]
 			)
 		);
+		$autoload = get_autoload_value( $autoload, $deprecated_keys[ $option ], $value );
```

Comment:
I don't think that's needed? It's a recursive call to `add_option()`, so we should use the same `$autoload` value that was passed and handle anything else inside the nested function execution (i.e. below in the code).
```suggestion
```